src/team/list.ts

```diff
@@ -0,0 +1,50 @@
+import { Message } from "discord.js";
+import { TeamArgs } from ".";
+import { listTeams } from "../db/teams";
+import { listTeamMemberships } from "../db/team_memberships";
+import { infoEmbed, errorEmbed } from "../util/embeds";
+
+export default async function list(
+  msg: Message,
+  args: TeamArgs
+): Promise<void> {
+  const page = parseInt(args.splitCommandLower[2]) || 1;
+  if (isNaN(page)) {
+    await msg.channel.send(
+      errorEmbed()
+        .setTitle("Invalid page number")
+        .setDescription("The page number you entered isn't a number.")
+    );
+    return;
+  }
+
+  const teams = await listTeams({
+    tournament_id: args.currentTournament.id,
+    meta: { limit: 10, offset: page, order_by: { exp: "id", dir: "DESC" } },
```

Comment:
```suggestion
    meta: { limit: 10, offset: ((page-1)*10), order_by: { exp: "id", dir: "DESC" } },
```
at page 1 offser should be 0 ((1-1)*10)=0, at 5 should be 40, etc.

Reply:
done, was going to ask that during call but forgot